experimental/SetPartitions/src/PartitionProperties.jl

```diff
@@ -0,0 +1,194 @@
+function size(p::SetPartition)
+    return length(p.lower_points) + length(p.upper_points)
+end
```

Comment:
I find all of these concrete implementations weird in this file and would personally look for them in the corresponding PartitionType.jl file instead.